### ISSUE TITLE
Exported cards' images and audio not synced in Ankidroid

### ISSUE THREAD

**Issue by sp3ctum** [lrorpilla/jidoujisho#6]
Here is an issue l just ran into.

How to reproduce:
- jidoujisho 0.13.0-beta
- latest ankidroid (2.14.6)
- desktop anki Version 2.1.22 (0ecc189a)

Steps:
- in jidoujisho, open a video and "export the current context to anki" to create a card
- in ankidroid, verify that the created card can display its image and play its audio
- in ankidroid, sync changes
- in desktop anki, likewise sync so the new card becomes available
- in desktop anki, Browse to see the card
- press Ctrl+L to verify that the card is missing its image, and can't play its audio

A workaround is to run "Check Media" in ankidroid before syncing. After this, the image and audio are synced just fine.

**Comment by sp3ctum** [comment]
I was able to reproduce this on my smartphone and my tablet.

Also did some investigation to see if this is anki related and came up with two findings:
- https://github.com/ankidroid/Anki-Android/issues/6485 Manually added audio files to collections.media don't get synced
- https://github.com/ankidroid/Anki-Android/pull/6641 A merged pull request that promises to "Allows adding media via the ankidroid-api"

Not being familiar with the code, maybe these are totally useless :)

**Comment by lrorpilla** [comment]
Yeah, this to me seems like AnkiDroid territory. The moment "export" is pressed, the image and audio file are stored in the AnkiDroid collection.media folder and the API is called to add the card to the user's database.

Shouldn't be an issue within this project, and seeing as Check Media solves it, seems to be all nominal.

Thanks for your report and for reproducing the issue, I'll add your workaround somewhere in documentation.

**Comment by lrorpilla** [comment]
I want to make a FAQ later onwards, but for now I've put a relevant line about Check Media under the `README.md` under *Exporting to AnkiDroid*.

**Comment by lrorpilla** [comment]
This is an old issue, but I have changed the way files are exported to the media collection in [**0.23.2**](https://github.com/lrorpilla/jidoujisho/releases/tag/0.23.2-beta), I believe AnkiDroid should now be aware of media files as they are imported into the collection, where beforehand I moved the files via the file system and didn't use their API. This means users probably no longer need to `Check Media` before syncing.

This also addresses an issue where users using SD cards for their AnkiDroid directory could not export files to the app, and removes the option to set the AnkiDroid directory (as the application is now aware of where it is without the user having to set it).
